roles/openshift_master/templates/master.yaml.v1.j2

```diff
@@ -101,7 +101,7 @@ projectConfig:
     mcsLabelsPerProject: 5
     uidAllocatorRange: 1000000000-1999999999/10000
 routingConfig:
-  subdomain: router.default.local
+  subdomain:  "{{ openshift.master.default_subdomain | default("") }}"
```

Comment:
@wshearn I thought about that, but router.default.local doesn't match up with the upstream default of router.default.svc.cluster.local